docs/lib/predictions/fragments/android/getting-started.md

```diff
@@ -0,0 +1,171 @@
+## Prerequisites
+
+* [Install and configure the Amplify CLI](~/cli/start/install.md)
+
+```bash
+npm install -g @aws-amplify/cli
+amplify configure
+```
+
+## Automated Configuration
```

Comment:
We'll have a Project Setup guide that will make this redundant shortly.

Reply:
will keep until it IS redundant